### ISSUE TITLE
misnamed-import warning for import A from '.'

### ISSUE THREAD

**Issue by devdoomari** [palantir/tslint#2878]
### Bug Report

- __TSLint version__: 5.4.2
- __TypeScript version__: 2.3.4
- __Running TSLint via__: (pick one) CLI and VSCode

#### TypeScript code being linted

```typescript
// the file is in: Logger folder, and '.' refers to index.ts file
import Logger, {
  LogItem,
} from '.';
```

error: Logger/logger.utils.ts
severity: 'Error'
message: 'Misnamed import. Import should be named '.' but found 'Logger' (import-name)'
at: '2,1'
source: 'tslint'

I think I've just stumbled on some edge case...
how about using directory name for import-name checking for this?

**Comment by ajafff** [comment]
That's the `import-name` rule of https://github.com/microsoft/tslint-microsoft-contrib.
You should probably open the issue there

**Comment by devdoomari** [comment]
@ajafff oh sorry closing!
